models/trello_schema.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+
+sources:
+  - name: trello_test
+    database: trello_test
+    description: This database contains relations created by tap-trello
+    schema: analytics
+    freshness: 
```

Comment:
We'll keep this simple for the time being, no need to include freshness or loaded_at_field in these table definitions. Just the name of each of the tables and it would be nice to clear up some of the empty space as well.

Reply:
okay. noted.

Reply:
okay